File: Solutions/589/java/program.java
// Copy from Quera

import java.util.Scanner;

public class Main {

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int n = scanner.nextInt();
        long fact = 1;
        for (int i = 0; i< n; i++)
            fact *= i+1;
        System.out.println(fact);
    }
}

File: Solutions/589/c++/program.cpp
// Copy from Quera

#include <iostream>
using namespace std;

int main()
{
    long long fact = 1;
    int n;
    cin >> n;
    for (int i = 0; i < n; i++)
        fact *= i + 1;
    cout << fact << "\n";
    return 0;
}